great_tables/_options.py

```diff
@@ -335,6 +336,10 @@ def tab_options(
     column_labels_hidden
         An option to hide the column labels. If providing `True` then the entire `column_labels`
         location won't be seen and the table header (if present) will collapse downward.
+    column_labels_units_pattern
```

Comment:
See comment on being unused. What happens if people set this? If it currently doesn't do anything then we need to set expectations.

Reply:
Now removed as it doesn't do anything now (with the other removals).